studies/templates/emails/study_announcement.html

```diff
@@ -0,0 +1,19 @@
+
+<p>Dear {{ user.nickname }},</p>
+
+<p>We're writing to invite you and your children to participate in the study "{{ study.name }}" that {{ study.lab.name }} is running on Lookit!</p>
+
+<ul>
+    <li>Who: <ul>{% for child in children %}<li>{{ child.given_name }}</li>{% endfor %}</ul></li>
```

Comment:
Is there some not-too-awkward thing to say after study.criteria to indicate the children listed might not be eligible? "This may include:"? I'm thinking of existing studies where criteria include currently being in the US, for instance (for IRB/funding reasons).